suffuse/inode_gen.go

```diff
@@ -0,0 +1,38 @@
+package suffuse
+
+type InodeGen struct {
+  fresh chan InodeNum
+}
+
+func NewRootInode(startInode uint64)*Inode {
```

Comment:
Naming and placement of this is not very intuitive. I think this method could be like:

```
func NewInode(num uint64 /* or even InodeNum */) *Inode
```

And I think it should be moved to the `Inode` file.

I might be totally wrong though. Maybe it should be `NewInodeGen` with the `NewInode` in the other file containing only `NewInodeGen(num).NextDir()` (splitting it into two methods).